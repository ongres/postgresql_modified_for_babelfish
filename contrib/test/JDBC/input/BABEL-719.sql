use master;
go

select $12.4524
go

select +$12.4524
go

select -$12.4524
go

select $+12.4524
go

select $-12.4524
go

select abs($12.4524)
go

select abs(+$12.4524)
go

select abs(-$12.4524)
go

select abs($+12.4524)
go

select abs($-12.4524)
go

-- udf accepting float
create function f719(@a float) returns int as begin return floor(@a) end
go

select f719($12.4524)
go

select f719(+$12.4524)
go

select f719(-$12.4524)
go

select f719($+12.4524)
go

select f719($-12.4524)
go

drop function f719
go
