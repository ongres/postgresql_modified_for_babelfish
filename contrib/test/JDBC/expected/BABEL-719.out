use master;
go

select $12.4524
go
~~START~~
numeric
12.4524
~~END~~


select +$12.4524
go
~~START~~
numeric
12.4524
~~END~~


select -$12.4524
go
~~START~~
numeric
-12.4524
~~END~~


select $+12.4524
go
~~START~~
numeric
12.4524
~~END~~


select $-12.4524
go
~~START~~
numeric
-12.4524
~~END~~


select abs($12.4524)
go
~~START~~
numeric
12.4524
~~END~~


select abs(+$12.4524)
go
~~START~~
numeric
12.4524
~~END~~


select abs(-$12.4524)
go
~~START~~
numeric
12.4524
~~END~~


select abs($+12.4524)
go
~~START~~
numeric
12.4524
~~END~~


select abs($-12.4524)
go
~~START~~
numeric
12.4524
~~END~~


-- udf accepting float
create function f719(@a float) returns int as begin return floor(@a) end
go

select f719($12.4524)
go
~~START~~
int
12
~~END~~


select f719(+$12.4524)
go
~~START~~
int
12
~~END~~


select f719(-$12.4524)
go
~~START~~
int
-13
~~END~~


select f719($+12.4524)
go
~~START~~
int
12
~~END~~


select f719($-12.4524)
go
~~START~~
int
-13
~~END~~


drop function f719
go
